apps/omg/lib/omg/state/core.ex

```diff
@@ -388,6 +425,44 @@ defmodule OMG.State.Core do
     }
   end
 
+  defp collect_fees(%Core{fees_paid: fees_paid} = state, tx_diffs) do
+    tx_fees =
+      tx_diffs
```

Comment:
rename `tx_diffs` to sth more specific... `token_surplussed` or `collectable_fee_per_token`.